### DISCUSSION TITLE
The CNAME demo.upptime.js.org is already taken

### DISCUSSION THREAD

**Discussion by stemy2** [upptime/upptime#129] [Q&A] [answered]
Hello,

I followed th instructions, but on my setting page, i see the following error: `The CNAME demo.upptime.js.org is already taken`. This is a second attempt after deletion of a misconfigured first attempt (i forgot to check "Include all branches" before validation). What shall i do now ?

**Comment by AnandChowdhary** [comment, marked as answer]
You should change the `cname` in your Upptime configuration to your domain, or remove it. Please see https://upptime.js.org/docs/get-started#update-configuration.
